### DISCUSSION TITLE
Left channel audio crackle

### DISCUSSION THREAD

**Discussion by Givingco** [FundamentalFrequency/.github#116] [Q&A]
Hey, I've been having a ton of fun with this thing but i was wondering if anyone else is having this issue. When I use some instruments, particularly the oscillator, the left audio channel has a lot of crackle if i'm playing a chord. I was just wondering if anyone has any ideas for a fix or any idea what's causing it because im pretty beginner when it comes to this kind of stuff. Thanks!

**Comment by iamdey** [comment]
I had this issue on both channels and I suspect my rasberry pi (2b) to be not enough powerful to handle it correctly. I had to play around with the OSC release and sustain to prevent it.
